java/src/main/java/be/tothepoint/model/projections/NrOfPlayersProjection.java

```diff
@@ -0,0 +1,22 @@
+package be.tothepoint.model.projections;
+
+import java.util.List;
+
+import be.tothepoint.model.Event;
+
+public class NrOfPlayersProjection implements Projection<Long> {
+
+  @Override
+  public Long project(List<Event> events) {
```

Comment:
@michelgrootjans do we want to include this projection?
Or maybe we want to rewrite it to use a simple for loop?
So people who don't know streams, maps,... can use for loops and are not confused with the syntax.

People who know map/filter/reduce will use it.